test/fs.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const metatests = require('metatests');
+const { mkdirp } = require('..');
+const fs = require('fs');
+const path = require('path');
+const pathName = 'test/ex1/ex2';
+
+const callback = msg => {
```

Comment:
Since this is the part of tests it should fail tests and not just output the error.